runtime/oak/src/lib.rs

```diff
@@ -1040,15 +1040,15 @@ pub struct ClosedCallFilter;
 impl Contains<Call> for ClosedCallFilter {
 	fn contains(c: &Call) -> bool {
 		match c {
-			Call::AutomationTime(_) => false,
-			Call::Balances(_) => false,
-			Call::Bounties(_) => false,
-			Call::Currencies(_) => false,
-			Call::ParachainStaking(_) => false,
-			Call::PolkadotXcm(_) => false,
-			Call::Treasury(_) => false,
-			Call::XTokens(_) => false,
-			Call::AutomationPrice(_) => false,
+			Call::AutomationTime(_) => true,
+			Call::Balances(_) => true,
```

Comment:
Keeping 10 decimlas for OAK. Is there an issue w/ KSM being 12 decimals and DOT being 10 decimals. @justinzhou93 to double check

Reply:
Just changes the token conversion rates.